doc-utils/server.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import http from "http";
+import express, { Express } from "express";
```

Comment:
Express ❤️

Men kan vi kalle disse for `devServer` og `devClient`, evt. `docServer` og `docClient` eller lignende? Har ikke så mye å si, føltes bare litt _vel_ generelt 😅

Reply:
Sure thing! Kan tydeliggjøre at dette er for utvikling og ikke prod.